tests/namespace_delete_test.go

```diff
@@ -82,16 +83,26 @@ func dynamicConfig() map[dynamicconfig.Key]interface{} {
 func (s *namespaceTestSuite) SetupSuite() {
 	s.logger = log.NewTestLogger()
 
-	clusterConfig, err := GetTestClusterConfig("testdata/integration_namespace_cluster.yaml")
-	s.Require().NoError(err)
+	var clusterConfig *TestClusterConfig
+	if TestFlags.PersistenceDriver == mysql.PluginNameV8 ||
+		TestFlags.PersistenceDriver == postgresql.PluginNameV12 ||
+		TestFlags.PersistenceDriver == sqlite.PluginName {
+		var err error
+		clusterConfig, err = GetTestClusterConfig("testdata/integration_test_cluster.yaml")
+		s.Require().NoError(err)
+	} else {
+		var err error
+		clusterConfig, err = GetTestClusterConfig("testdata/integration_namespace_cluster.yaml")
```

Comment:
`integration_namespace_cluster` is a weird name and I have no idea what it means. is there a more descriptive one?

Reply:
ok, I clean it up.